Makefile

```diff
@@ -1,12 +1,12 @@
 test: .build-tmp/build-image src/* test/*
-	docker run --rm -it -v ${PWD}/src:/app/src:ro -v ${PWD}/test:/app/test:ro rgpm:test
+	docker run --rm -it --cap-add=SYS_ADMIN -v ${PWD}/src:/app/src:ro -v ${PWD}/test:/app/test:ro rgpm:test
```

Comment:
This is where the SYS_ADMIN privilege is added. It is required to run Chrome headless